Tasks/Karachev/Task7/bl.py

```diff
@@ -0,0 +1,43 @@
+import data
+
+def show_animals():
+    animals = data.show_all_animals()
+    return '\n'.join([f"{c[0]}: color - {c[1]}, code - {c[2]}, price - {c[3]} byn, amount - {c[4]} pcs" for c in animals])
+
+
+def show_pet_food():
+    food = data.show_all_pet_food()
+    return '\n'.join([f"{c[0]}: weight - {c[1]}, code - {c[2]}, price - {c[3]} byn, amount - {c[4]} pcs" for c in food])
```

Comment:
hard coded categories is not a good option, you should opt for a function that will provided any category based on a parameter